packages/core/contracts-ovm/insured-bridge/implementation/BridgeRouter.sol

```diff
@@ -62,8 +62,8 @@ contract BridgeRouter is OVM_CrossDomainEnabled {
     // Associates each deposit with a unique ID.
     mapping(uint256 => Deposit) public deposits;
     // If a deposit is disputed, it is removed from the `deposits` mapping and added to the `disputedDeposits` mapping.
-    // There can only be one disputed deposit for each deposit ID.
-    mapping(uint256 => Deposit) public disputedDeposits;
+    // There can only be one disputed deposit per relayer for each deposit ID.
+    mapping(uint256 => mapping(address => Deposit)) public disputedDeposits;
```

Comment:
@chrismaree is this how you envisioned the `disputedDeposits` struct looking?

Reply:
ye, that seems reasnoble. Prehaps add a comment to explain the mapping. something like:
`depositId->disputer->deposit`